client/blocks/reader-post-card/byline.jsx

```diff
@@ -79,7 +80,7 @@ class PostByline extends React.Component {
 							author={ post.author }
 							siteUrl={ streamUrl }
 							post={ post }>
-							{ post.author.name }
+							{ decodeEntities( post.author.name ) }
```

Comment:
I think that comes at a pretty hefty download cost though. how's the bundle size look before and after?